src/main/fc/runtime_config.c

```diff
@@ -32,7 +32,11 @@ static uint32_t enabledSensors = 0;
 const char *armingDisableFlagNames[]= {
     "NOGYRO", "FAILSAFE", "RXLOSS", "BADRX", "BOXFAILSAFE",
     "THROTTLE", "ANGLE", "BOOTGRACE", "NOPREARM", "LOAD",
-    "CALIB", "CLI", "CMS", "OSD", "BST", "MSP", "ARMSWITCH"
+    "CALIB", "CLI", "CMS", "OSD", "BST", "MSP",
+#ifdef USE_RUNAWAY_TAKEOFF
```

Comment:
This should not be conditional. If it is, the numbering of flags after it will become target dependent, creating problems for interfacing apps and documentation.